require.js

```diff
@@ -8,10 +8,8 @@
   var cache = {};
   var aliases = {};
   var has = ({}).hasOwnProperty;
-
-  var endsWith = function(str, suffix) {
-    return str.indexOf(suffix, str.length - suffix.length) !== -1;
-  };
+  var extExp = /\.[^\/]+$/
+  var indexExp = /\/index(\.[^\/]+)?$/
```

Comment:
Can you please fix this to have `[^\/^.]` instead of `[^\/]` — we want 'react/lib/Object.assign.js` to be aliased as `react/lib/Object.assign`, not `react/lib/Object`